.agents/skills/live-robot-telemetry/scripts/nt_live_common.py

```
@@ -0,0 +1,126 @@
+        root / f"maven/edu/wpi/first/wpiutil/wpiutil-java/{version}/wpiutil-java-{version}.jar",
+    ]
+    missing = [str(jar) for jar in jars if not jar.exists()]
+    if missing:
+        raise SystemExit("Missing WPILib Java dependency jars:\n" + "\n".join(missing))
+    return os.pathsep.join(str(jar) for jar in jars)
+
+
+def extract_natives(root: Path, target: Path) -> Path:
+    """Extract WPILib native libraries to a temporary directory."""
+    version = wpilib_version(root, "edu/wpi/first/ntcore", "ntcore-cpp")
+    system = platform.system().lower()
```

Ruff (0.16.0)

[error] Undefined name `platform`

(F821)